app/experimenter/experiments/constants/nimbus.py

```diff
@@ -175,6 +175,17 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_MOBILE_FIRST_RUN = NimbusTargetingConfig(
+    name="First start-up users",
+    slug="first_run_mobile",
+    description=(
+        "First start-up users (i.e first run after installing the app) on mobile"
+    ),
+    targeting=("isFirstRun == 'true'"),
+    desktop_telemetry="",
```

Comment:
I'm not 100% sure of this, since this is the first special targeting parameter on mobile...

Reply:
Yeah you should be able to leave that blank.